src/components/utils/AssetValueFormatter/index.tsx

```diff
@@ -0,0 +1,36 @@
+import Decimal from "decimal.js";
+import { Token } from "types/api";
+import { amountToValue, formatDecimals, formatNumber } from "utils";
+import { Numeric } from "@xpla/xpla.js";
+import OverflowTooltip from "../OverflowTooltip";
+
+interface AssetValueFormatterProps {
+  asset?: Partial<Pick<Token, "decimals" | "symbol">>;
+  amount?: Decimal.Value;
+  showSymbol?: boolean;
+}
+
+function AssetValueFormatter({
+  asset,
+  amount,
+  showSymbol = true,
+}: AssetValueFormatterProps) {
+  const parsedValue = Numeric.parse(
+    amountToValue(amount, asset?.decimals) || "0",
+  );
+  const formattedValue = formatNumber(formatDecimals(parsedValue, 3));
+  const valueToDisplay =
+    parsedValue.lt(0.001) && !parsedValue.eq(0) ? "< 0.001" : formattedValue;
+  return (
+    <OverflowTooltip
+      content={`${parsedValue.toFixed()}${
+        showSymbol ? ` ${asset?.symbol || ""}` : ""
+      }`}
+      disabled={valueToDisplay !== formattedValue ? false : undefined}
+    >
```

Comment:
It's not a fatal problem, but I have one suggestion.

how about this one
 <in this file>
`disabled={valueToDisplay !== formattedValue}`
<When you use it>
` <Tooltip
      ref={tooltipRef}
      disabled={**disabledFromProps? disabledFromProps : disabled**}
      content={content || children}
      {...props}
    >
      {typeof children === "string" ? <span>{children}</span> : children}
    </Tooltip>`

Because 
1. disalbed usually has a boolean value.
2. You are of course using typescript, but undefind looks uneasy because it is a falsy value in javascript. 
3. The number of codes is decreasing

If you have a reason to use undefind, please share it with me
thank you for reading